src/ui/controls/RadioListControl.tsx

```diff
@@ -40,6 +40,15 @@ export class RadioListControl extends React.Component<IStringControlProps, void>
   }
 
   /** @override */
```

Comment:
Are the `@override`s necessary?  React components are expected to implement these methods.